Dockerfile

```diff
@@ -46,13 +46,17 @@ ENV STATE_DB_IMPL="geth"
 ENV VM_IMPL="geth"
 ENV LD_LIBRARY_PATH=./
 
-EXPOSE 5050
-EXPOSE 6060
-EXPOSE 18545
-EXPOSE 18546
+EXPOSE 5050 6060 18545 18546
```

Comment:
is not it the same? We should keep code formatting for later.

Reply:
It's the same. Will include this in revert

Reply:
Change removed